uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/annual-entitlements.component.js

```diff
@@ -79,18 +79,25 @@
       }
 
       /**
-        * Returns the URL to the Manage Entitlement page.
-        *
-        * The given contact ID is added to the URL, as the cid parameter.
-        *
-        * @param  {Number} contactId
-        * @return {String}
-        */
-      function getEditEntitlementsPageURL (contactId) {
+       * Gets the URL to the Manage Entitlement page
+       * for the contact for an absence period with a provided ID.
```

Comment:
"for the contact, absence period, and absence type provided"

It would also be nice to mention that it configures the return URL to the current one.